Решение финального задания спринта 9

IGrigoryRI opened the pull request.

Commit:
Решение финального задания спринта 9

Commit:
Решение финального задания спринта 9

kriwsha commented on src/server/handlers/EpicHandler.java:
```diff
@@ -0,0 +1,130 @@
+        String method = exchange.getRequestMethod();
+
+        switch (method) {
+            case "GET":
```
Было бы здорово вынести все эти названия методов в отдельные константы, чтобы во всем коде использовались только эти константы. Так будет меньше вероятности ошибок

kriwsha commented on src/server/HttpTaskServer.java:
```diff
@@ -0,0 +1,39 @@
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+public class HttpTaskServer {
```
Не забывайте документировать код

Commit:
Решение финального задания спринта 9

Добавлен enum класс "HttpMethod" для хранения названий методов в виде констант
Добавлено документирование кода
